tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -59,27 +59,18 @@ build_rootfs() {
 	fi
 	# trim whitespace
 	PACKAGES=$(echo $PACKAGES |xargs )
-	EXTRA_PKGS=$(echo $EXTRA_PKGS |xargs)
 	# add comma as debootstrap needs , separated package names.
 	# Don't change $PACKAGES in config.sh to include ','
 	# This is done to maintain consistency
 	PACKAGES=$(echo $PACKAGES | sed  -e 's/ /,/g' )
-	EXTRA_PKGS=$(echo $EXTRA_PKGS | sed  -e 's/ /,/g' )
-
-	# extra packages are added to packages and finally passed to debootstrap
-	if [ "${EXTRA_PKGS}" = ""  ]; then
-		echo "no extra packages"
-	else
-		PACKAGES="${PACKAGES},${EXTRA_PKGS}"
-	fi
 
 	${PKG_MANAGER} --variant=minbase \
 		--arch=${ARCHITECTURE}\
 		--include="$PACKAGES" \
 		${OS_NAME} \
 		${ROOTFS_DIR}
 
-	chroot $ROOTFS_DIR ln -s /lib/systemd/systemd /usr/lib/systemd/systemd
+	[ -n "${EXTRA_PKGS}" ] && chroot $ROOTFS_DIR apt-get install -y ${EXTRA_PKGS}
```

Comment:
@snir911 almost LGTM -- technically, this line does not substitute the other line like the diff implies, but the added line belongs to the new installation method of chrony (which you want to backport) and the removed line belongs to the upgrade to 20.04 (which you didn't want to backport).

...I'm not happy with my phrasing. I think it should just be:

```sh
chroot $ROOTFS_DIR ln -s /lib/systemd/systemd /usr/lib/systemd/systemd
[ -n "${EXTRA_PKGS}" ] && chroot $ROOTFS_DIR apt-get install -y ${EXTRA_PKGS}
```